ee/server/service/scep_proxy_test.go

```diff
@@ -44,6 +50,19 @@ func TestValidateNDESSCEPAdminURL(t *testing.T) {
 	returnStatus = http.StatusNotFound
 	err := ValidateNDESSCEPAdminURL(context.Background(), proxy)
 	assert.ErrorContains(t, err, "unexpected status code")
+	returnStatus = http.StatusOK
+
+	// Catch timeout issue
+	origNDESTimeout := NDESTimeout
+	NDESTimeout = ptr.Duration(1 * time.Microsecond)
+	t.Cleanup(func() {
+		NDESTimeout = origNDESTimeout
+	})
+	wait = true
+	err = ValidateNDESSCEPAdminURL(context.Background(), proxy)
+	assert.ErrorIs(t, err, context.DeadlineExceeded)
+	wait = false
```

Comment:
Yes, I always run locally with `-race` (not `--race`)

Reply:
:+1: awesome  (fwiw, Go supports the double-dashes too, and since that's so common for other commands that's my muscle memory adding two when it's a long-form flag :D )